airflow-bi/Makefile

```
@@ -7,6 +7,7 @@ all: build push
 
 .PHONY: build
 build:
+	poetry export -f requirements.txt --output requirements.txt
```

```suggestion
	poetry export --output requirements.txt
```

`requirements.txt` is the default.